README.md

```diff
@@ -1,19 +1,27 @@
 # machine-config-operator
 
-## machine-config-operator
-- Build: `WHAT=machine-config-operator ./hack/build-go.sh`
+## Generic building
 
-## machine-config-server
+You can either use the Makefile or the `hack/build-go.sh` script directory to build the targets. When using `hack/build-go.sh` you will need to run it via `WHAT=<TARGET> hack/build-go.sh`
```

Comment:
Hmm, not sure it's worth documenting both if the goal is to simplify things. Let's just document the `make` way?

Reply:
> Hmm, not sure it's worth documenting both if the goal is to simplify things. Let's just document the make way?

Any thoughts about this?

Reply:
Removed, an only documented the Makefile convetion.